src/number_theory/padics/padic_norm.lean

```diff
@@ -37,25 +37,22 @@ by taking `[fact (prime p)]` as a type class argument.
 p-adic, p adic, padic, norm, valuation
 -/
 
-/--
-If `q ≠ 0`, the p-adic norm of a rational `q` is `p ^ (-(padic_val_rat p q))`.
-If `q = 0`, the p-adic norm of `q` is 0.
--/
-def padic_norm (p : ℕ) (q : ℚ) : ℚ :=
-if q = 0 then 0 else (↑p : ℚ) ^ (-(padic_val_rat p q))
+/-- If `q ≠ 0`, the `p`-adic norm of a rational `q` is `p ^ -padic_val_rat p q`.
```

Comment:
Yet another thing I don't think we have an explicit rule for, but for which I'd love to see one.